tests/org.genivi.faracon.tests/src/org/genivi/faracon/tests/aspects_on_file_level/FilePathsHelperTest.xtend

```diff
@@ -41,24 +41,39 @@ class FilePathsHelperTest {
 		val childPath = "src/org/genivi/faracon/tests/aspects_on_franca_methods/a2f"
 		val childFile = "src/org/genivi/faracon/tests/aspects_on_franca_methods/a2f/broadcastArgumentExpected.fidl"
 		executeFilePathHelperTest(#[childFile, childPath, path],
-			#[path + "a2f/broadcastArgumentExpected.fidl", path + "a2f/multipleMethodInputArgumentsExpected.fidl",
-				path + "a2f/inOutExpected.fidl", path + "a2f/multipleMethodReturnValuesExpected.fidl",
-				path + "f2a/arrayMethodInputArgument.fidl", path + "f2a/broadcastArgument.fidl",
-				path + "f2a/fireAndForgetMethod.fidl", path + "f2a/methodWithInAndOutArguments.fidl",
-				path + "f2a/multipleMethodInputArguments.fidl", path + "f2a/multipleMethodReturnValues.fidl",
-				path + "f2a/oneMethodInputArgument.fidl", path + "f2a/oneMethodReturnValue.fidl"])
+			#["/a2f/broadcastArgumentExpected.fidl", "/broadcastArgumentExpected.fidl",
+				"/a2f/multipleMethodInputArgumentsExpected.fidl", "/a2f/inOutExpected.fidl",
+				"/a2f/multipleMethodReturnValuesExpected.fidl", "/f2a/arrayMethodInputArgument.fidl",
+				"/f2a/broadcastArgument.fidl", "/f2a/fireAndForgetMethod.fidl", "/f2a/methodWithInAndOutArguments.fidl",
+				"/f2a/multipleMethodInputArguments.fidl", "/f2a/multipleMethodReturnValues.fidl",
+				"/f2a/oneMethodInputArgument.fidl", "/f2a/oneMethodReturnValue.fidl", "/inOutExpected.fidl",
+				"/multipleMethodInputArgumentsExpected.fidl", "/multipleMethodReturnValuesExpected.fidl"])
+	}
+
+	@Test
+	def void testIdentivalFilePathAsInput() {
+		val path = "src/org/genivi/faracon/tests/aspects_on_franca_methods/"
+		val identicalPath = "src/org/genivi/faracon/tests/aspects_on_franca_methods/"
+		executeFilePathHelperTest(#[path, identicalPath],
+			#["/a2f/broadcastArgumentExpected.fidl", "/a2f/multipleMethodInputArgumentsExpected.fidl",
+				"/a2f/inOutExpected.fidl", "/a2f/multipleMethodReturnValuesExpected.fidl",
+				"/f2a/arrayMethodInputArgument.fidl", "/f2a/broadcastArgument.fidl", "/f2a/fireAndForgetMethod.fidl",
+				"/f2a/methodWithInAndOutArguments.fidl", "/f2a/multipleMethodInputArguments.fidl",
+				"/f2a/multipleMethodReturnValues.fidl", "/f2a/oneMethodInputArgument.fidl",
+				"/f2a/oneMethodReturnValue.fidl"])
 	}
 
 	def private void executeFilePathHelperTest(String[] paths, Collection<String> expectedResult) {
 		// given: the path
 		// when
-		var foundFilePaths = FilePathsHelper.findFiles(paths, "fidl")
+		val foundFilePaths = FilePathsHelper.findInputFiles(paths, "fidl")
 
+		var relativeFilePaths = foundFilePaths.map[it.absolutePath.substring(it.basePathLength)]
```

Comment:
substring(it.basePathLength) -> substring(it.basePathLength + 1)

Reply:
done